src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
```

Comment:
I don't really know what this field does, and since it isn't really related to the PR I'm going to leave it as is for now.  If you want to track it down and get it changed, we'd certainly appreciate the assist.